opennms-base-assembly/src/main/filtered/etc/collectd-configuration.xml

```diff
@@ -137,16 +153,18 @@
     </package>
 
 
-    <collector service="JMX-Cassandra" class-name="org.opennms.netmgt.collectd.Jsr160Collector"/>
-    <collector service="JMX-Cassandra-Newts" class-name="org.opennms.netmgt.collectd.Jsr160Collector"/>
+    <collector service="PostgreSQL" class-name="org.opennms.netmgt.collectd.JdbcCollector"/>
     <collector service="SNMP" class-name="org.opennms.netmgt.collectd.SnmpCollector"/>
     <collector service="WMI" class-name="org.opennms.netmgt.collectd.WmiCollector"/>
     <collector service="WS-Man" class-name="org.opennms.netmgt.collectd.WsManCollector"/>
-    <collector service="OpenNMS-JVM" class-name="org.opennms.netmgt.collectd.Jsr160Collector"/>
-    <collector service="Minion-JVM" class-name="org.opennms.netmgt.collectd.Jsr160Collector"/>
     <collector service="VMware-VirtualMachine" class-name="org.opennms.netmgt.collectd.VmwareCollector"/>
     <collector service="VMware-HostSystem" class-name="org.opennms.netmgt.collectd.VmwareCollector"/>
     <collector service="VMwareCim-HostSystem" class-name="org.opennms.netmgt.collectd.VmwareCimCollector"/>
-    <collector service="PostgreSQL" class-name="org.opennms.netmgt.collectd.JdbcCollector"/>
-</collectd-configuration>
 
+    <!-- TODO: Rename to 'JMX-OpenNMS' -->
```

Comment:
I'm going to leave this here for now because I think (for consistency's sake) that we should address this in 19 but it will require an upgrade script that can update the database.